prime-dotnet-webapi/Controllers/LabtechsController.cs

```diff
@@ -0,0 +1,60 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+using Prime.Auth;
+using Prime.Models;
+using Prime.Models.Api;
+using Prime.Services;
+using Prime.ViewModels;
+using Prime.ViewModels.Labtech;
+using Prime.HttpClients;
+
+namespace Prime.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/parties/[controller]")]
+    [ApiController]
+    [Authorize(Policy = Policies.User)]
+    public class LabtechsController : ControllerBase
+    {
+        private readonly IPartyService _partyService;
+        private readonly IKeycloakAdministrationClient _keycloakClient;
+
+        public LabtechsController(
+            IPartyService partyService,
+            IKeycloakAdministrationClient keycloakClient)
+        {
+            _partyService = partyService;
+            _keycloakClient = keycloakClient;
+        }
+
+        // POST: api/parties/labtechs
+        /// <summary>
+        /// Creates a new Labtech.
+        /// If successful, also updates Keycloak with additional user info and the Labtech role.
+        /// </summary>
+        [HttpPost(Name = nameof(CreateLabtech))]
+        [ProducesResponseType(typeof(ApiBadRequestResponse), StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        public async Task<ActionResult> CreateLabtech(LabtechCreateModel labtech)
+        {
+            if(await _partyService.UserIdExistsAsync(User.GetPrimeUserId()))
+            {
+                ModelState.AddModelError("Labtech.UserId", "A Labtech already exists with this User Id.");
+                return BadRequest(ApiResponse.BadRequest(ModelState));
+            }
+
+            Party model = null; //TODOD: temp Labtech.From(labtech, User);
+            await _partyService.CreatePartyAsync(model);
+
```

Comment:
`labtech` variable seems properly populated but why are we passing in null for the `model` variable?  System.ArgumentNullException gets thrown